lib/rest-api.php

```diff
@@ -307,6 +307,43 @@ function gutenberg_register_taxonomy_prepare_functions( $taxonomy ) {
 }
 add_filter( 'registered_taxonomy', 'gutenberg_register_taxonomy_prepare_functions' );
 
+/**
+ * Ensure that the wp-json index contains the 'theme-supports' setting as
+ * part of its site info elements.
+ *
+ * @see https://core.trac.wordpress.org/ticket/45016
```

Comment:
Based on recent discussion in the ticket, it appears the current approach for these details is via a `?status=active` parameter on the `/wp/v2/themes` endpoint, not exposing on the root.

Reply:
I wondered what was going on, because I had to add this in two places for it to show up everywhere. Do I just need to remove it from here?

Reply:
I assume we'd need to also bring in any necessary revisions to support the `status` parameter as well.

Reply:
Could that be done as a separate PR? I'm a little concerned that the discussion on that ticket is still ongoing, and I don't want to delay getting this fix out to the peeps who are having theme conflicts.

Reply:
Just to clarify, the changes on 45016 were merged into Gutenberg before they were committed to core. See #10518